wireguard-vpn/js/MainController.js

```diff
@@ -404,18 +412,27 @@ Ext.define('Ung.apps.wireguard-vpn.cmp.WireGuardVpnTunnelRecordEditorController'
     // get next pool address
     getNextUnusedPoolAddr: function(){
```

Comment:
Isn't this function and function on line number 305 same with same function body ?

Reply:
Only difference seems to be the store and addressPool variables.
We should refactor the common code and parametrize the changing values.